src/main/java/seedu/address/logic/commands/ChangePasswordCommand.java

```diff
@@ -0,0 +1,139 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONFIRM_PASSWORD;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NEW_PASSWORD;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PASSWORD;
+
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+
+import com.google.common.hash.Hashing;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.UserPrefs;
+
+//@@author charlesgoh
+/**
+ * Changes user's password provided old password is correct and new passwords match.
+ */
+public class ChangePasswordCommand extends Command {
+    public static final String COMMAND_WORD = "changepassword";
+    public static final String COMMAND_ALIAS = "cps";
+
+    public static final String MESSAGE_SUCCESS = "Password changed successfully";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Changes user password. \n"
+            + "[" + PREFIX_PASSWORD + "PASSWORD] "
```

Comment:
It seems that all the fields are required, so they should not have [ ] around them.

Reply:
Nice spot 👍